.circleci/config.yml

```diff
@@ -3,10 +3,10 @@
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
 #
 defaults: &defaults
-  working_directory: /tmp/repo
-  docker:
-    # specify the version you desire here
-    - image: circleci/node:8.12.0-browsers
+    working_directory: /tmp/repo
+    docker:
+        # specify the version you desire here
+        - image: circleci/node:8.12.0-browsers
```

Comment:
was the indentation incorrect before?

Reply:
Indentation was reverted back to original.